### PR TITLE
Feature/get active tickets

### PR DESCRIPTION
**Pull request by mfbmina**
Create two new request that get all active tickets of a server.

This server can be a bm or a vm.

### CODE REVIEW

File: lib/fog/softlayer/requests/compute/get_virtual_guest_active_tickets.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# LICENSE: MIT (http://opensource.org/licenses/MIT)
+#
+module Fog
+  module Compute
+    class Softlayer
+      class Mock
+        # Gets all Virtual Guest active tickets
+        # @param [Integer] id
+        # @return [Excon::Response]
+        def get_virtual_guest_active_tickets(id)
```
Comment (fernandes) [inline lib/fog/softlayer/requests/compute/get_virtual_guest_active_tickets.rb]: here is not returning a ticket...

File: lib/fog/softlayer/requests/compute/get_bare_metal_active_tickets.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+module Fog
+  module Compute
+    class Softlayer
+      class Mock
+        # Gets all Bare Metal active tickets
+        # @param [Integer] id
+        # @return [Excon::Response]
+        def get_bare_metal_active_tickets(id)
+          response = Excon::Response.new
+          found = self.get_bare_metal_servers.body.map{|server| server['id']}.include?(id)
```
Comment (fernandes) [inline lib/fog/softlayer/requests/compute/get_bare_metal_active_tickets.rb]: here is not returning a ticket...

**Comment by fernandes** [comment]
doc examples please :)

**Comment by mfbmina** [comment]
Done @fernandes  :smile:

**Comment by fernandes** [comment]
insanely great! good job @mfbmina !

**Pull request merged**